packages/react-components/react-icons-compat/src/GlobalSettings.ts

```diff
@@ -0,0 +1,122 @@
+import { getWindow } from './getWindow';
+
+/**
+ * Storing global state in local module variables has issues when more than one copy
+ * if the module gets loaded on the page (due to a bundling error or simply by consuming
+ * a prebundled script.)
+ *
+ * This file contains helpers to deal with the getting and setting local state, and allows
+ * callers to get called back when it mutates.
+ */
+
+const GLOBAL_SETTINGS_PROP_NAME = '__globalSettings__';
+const CALLBACK_STATE_PROP_NAME = '__callbacks__';
+
+let _counter = 0;
+
+/**
+ * Change description used for change callbacks in GlobalSettings.
+ *
+ * @public
+ * {@docCategory IChangeDescription}
+ */
+export interface IChangeDescription {
```

Comment:
We have dropped the `I` prefix in all v9 packages.

```suggestion
export interface ChangeDescription {
```